File: content-solutions/backtracking/team-picks.py
def solution(n, k):
    result = []
    path = []

    def pick(start):
        if len(path) == k:
            result.append(path[:])
            return
        for i in range(start, n + 1):
            path.append(i)
            pick(i + 1)
            path.pop()

    pick(1)
    return result

File: content-solutions/backtracking/team-picks.js
function solution(n, k) {
  const result = [];
  const path = [];
  function pick(start) {
    if (path.length === k) {
      result.push([...path]);
      return;
    }
    for (let i = start; i <= n; i += 1) {
      path.push(i);
      pick(i + 1);
      path.pop();
    }
  }
  pick(1);
  return result;
}
